src/components/AgreementSections/Info.js

```diff
@@ -28,7 +28,7 @@ export default class Info extends React.Component {
 
   render() {
     const { agreement } = this.props;
-
+    const agreementIsClosed = (agreement.agreementStatus !== null && (agreement.agreementStatus.value === 'closed'));
```

Comment:
Same thing as above, an entry for `closed` in `ui-agreements/src/constants/statuses.js` would be nicer.

Reply:
Done as part of same commit as above